clang 3.8 : compile fail

src/lib_json/json_tool.h:66:18: error: implicit conversion changes signedness: 'signed char' to 'char' [-Werror,-Wsign-conversion]
    *--current = static_cast<signed char>(value % 10U + static_cast<unsigned>('0'));
               ~ ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

--- a/src/lib_json/json_tool.h
+++ b/src/lib_json/json_tool.h
@@ -63,7 +63,7 @@ typedef char UIntToStringBuffer[uintToStringBufferSize];
 static inline void uintToString(LargestUInt value, char*& current) {
   *--current = 0;
   do {
-    *--current = static_cast<signed char>(value % 10U + static_cast<unsigned>('0'));
+    *--current = static_cast<char>(value % 10U + static_cast<unsigned>('0'));
     value /= 10;
   } while (value != 0);
 }

